pages/spicedb/concepts/datastore-migrations.mdx

```diff
@@ -16,82 +16,52 @@ For example, releases that include changes to datastore can require that users u
-This document's focus is on the former; the latter is documented [here].
-
-[here]: ../modeling/migrating-schema
-
-### SpiceDB migration tooling
-
-SpiceDB ships with a migration command: `spicedb migrate`.
-This command powers all of Authzed products' zero down-time migrations, so it's guaranteed to be battle-tested and supportable.
-And while you are free to explore other tools to help you migrate, we cannot recommend them.
-
-If you do not care about causing downtime or you are bringing up a new cluster, you can always run the following command to migrate:
-
-```
-spicedb migrate head
-```
-
-In most cases, this command will not actually cause downtime, but one should confirm that's the case before executing on production environments with uptime requirements.
+In most cases, this command will not actually cause downtime, but one should confirm that's the case in a non-production environment before executing on production environments with uptime requirements.
 
-## DB Migration Compatibility
+## Migration Compatibility
 
-On startup, SpiceDB checks to see whether its desired DB migration tag matches the DB migration tag held in the database.
-If they differ, the instance will error out, which prevents an instance from coming up when it can't run against the current datastore DDL.
-
-SpiceDB only performs this check on startup, which means that a new DB migration won't cause existing SpiceDB instances to break (unless they restart for other reasons),
+On startup, SpiceDB checks to see whether its desired datastore migration tag matches the migration tag held in the datastore itself.
+If they differ, SpiceDB will error out. SpiceDB only performs this check on startup, which means that a new datastore migration won't cause existing SpiceDB instances to break (unless they restart for other reasons),
 provided that the DDL is compatible with the existing instance.
 
-The information in the Upgrade Notes of a given release can help determine whether the DDL will be compatible between two versions.
-Using the SpiceDB Operator is the easiest way to ensure that all DB migrations are compatible and applied correctly.
+SpiceDB strives to main compatibility across each version and its following minor version. When doing a major version update, we recommend reading the Upgrade Notes of the target release to see if any additional steps are required on your side.
+
+Using the [SpiceDB Operator] is the easiest way to ensure that all datastore migrations are compatible and applied correctly.
 
 ### Overriding Migration Compatibility
 
-Under some circumstances, you may want to run a version of SpiceDB against a DB migration other than the one it's expecting.
-If you know that the version of SpiceDB you want to run is compatible with the DB migration, you can use the `--datastore-allowed-migrations`
+Under some circumstances, you may want to run a version of SpiceDB against a datastore migration other than the one it's expecting.
```

Comment:
we seem to have settled on the term `datastore` instead of `database`. i have my reservations on that decision, but we should be consistent.